common/ailments/index.ts

```diff
@@ -16,7 +17,8 @@ const cardClasses: Array<Ailment> = [
     new SlownessAilment(),
     new WeaknessAilment(),
     new ProtectedAilment(),
-    new DyedAilment()
+    new DyedAilment(),
+    new MuseumCollectionAilment()
```

Comment:
We tend to leave a trailing comma so when we add new stuff only one line is changed, this can be left as is for now though